mysql-test/suite/innodb/r/innodb.result

```diff
@@ -3390,3 +3390,9 @@ Table	Op	Msg_type	Msg_text
 test.t1	check	status	OK
 ALTER TABLE t1 FORCE;
 DROP TABLE t1;
+SET GLOBAL innodb_buffer_pool_size = (256 + 64) * @@innodb_page_size -1;
+ERROR 42000: Variable 'innodb_buffer_pool_size' can't be set to the value of '5242879'
+SHOW WARNINGS;
+Level	Code	Message
+Warning	1210	innodb_buffer_pool_size must be at least 5242880 for innodb_page_size=16384
+Error	1231	Variable 'innodb_buffer_pool_size' can't be set to the value of '5242879'
```

Comment:
This test would be better added to a file that uses `innodb_page_size.inc` so that it will be tested with all 5 page sizes.